scripts/ci/install-rust.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+# Install common packages
+apt-get update
+apt-get install --no-install-recommends -y clang musl-tools
+rm -rf /var/lib/apt/lists/*
```

Comment:
This `rm` is causing issues... Maybe these `apt` lines aren't needed?

Reply:
I think we need `clang` but maybe not `musl-tools`...

Reply:
Confirmed, `clang` is installed in the step before this script is called. `musl-tools` is never installed because we use a different approach to get the musl libc for x-compilation.

Reply:
Awesome. Thanks! Pushed the change.